docs/source/daml/reference/interfaces.rst

```diff
@@ -85,8 +79,7 @@ Interface Choices
    :start-after: -- INTERFACE_CHOICES_BEGIN
    :end-before: -- INTERFACE_CHOICES_END
 
-- Interface choices work in a very similar way to template choices. Any contract
-  of an implementing interface will grant the choice to the controlling party.
+- Interface choices work in a very similar way to template choices. Any contract of a template type that is an ``interface instance`` of the interface will grant the choice to the controlling party.
```

Comment:
I get the point of the "is a" relationship, but I think it's worth considering the idea of giving more emphasis to the "has a" relationship, given that these might also be defined retroactively. What do you think about this?
```suggestion
- Interface choices work in a very similar way to template choices. Any contract of a template type for which an interface instance exists, will grant the choice to the controlling party.
```

Reply:
thank you, I think it works much better with "has a" language